packages/server/yugabyted.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Daemon to start a single-node YugabyteDB process
+Documentation=https://docs.yugabyte.com
+
+[Service]
+Type=forking
+User=yugabyte
+ExecStart=/usr/bin/python /opt/yugabytedb/bin/yugabyted start --config /etc/yugabytedb/yugabytedb.conf
+PIDFile=/opt/yugabytedb/yugabyted.pid
```

Comment:
I think this pid file location is going to move to the data directory /var/lib/yugabytedb/yugabyte-data/ in the next release. we don't need to update it right now but just an fyi.